distrobox-init

```diff
@@ -260,6 +269,32 @@ mount_bind() (
 	return 0
 )
 
+# Extract soft path from full host library path
+# Arguments:
+#   source_dir
+# Outputs:
+#   Path in the guest container where to put the lib
+get_nvidia_lib_path() {
+	source_dir="$1"
+
+	# Some Debian based systems use this type of path
+	if [ -e "/run/host/usr/lib/x86_64-linux-gnu/" ]; then
+		dest_dir="$(printf "%s" "${source_dir}" | sed 's|/run/host/usr/lib/x86_64-linux-gnu||g')"
+		echo "${dest_dir}"
+		return
+	# /usr/lib64 is quite popular on rpm based distros
+	elif [ -e "/run/host/usr/lib64" ]; then
+		dest_dir="$(printf "%s" "${source_dir}" | sed 's|/run/host/usr/lib64||g')"
+		echo "${dest_dir}"
+		return
+	# Fallback to /usr/lib if none of the previous are present
+	else
+		dest_dir="$(printf "%s" "${source_dir}" | sed 's|/run/host/usr/lib||g')"
+		echo "${dest_dir}"
+		return
+	fi
```

Comment:
```suggestion
	else
		dest_dir="$(printf "%s" "${source_dir}" | sed 's|/run/host/usr/lib64||g' | sed 's|/run/host/usr/lib||g')"
		echo "${dest_dir}"
		return
	fi
```

On arch linux the libraries are in `/usr/lib` instead of `/usr/lib64`, however `/usr/lib64` still exists, meaning the mounting would not work properly. This would fix it by just not checking wether to use `/usr/lib` or `/usr/lib64` and uses `sed` for both directories.

Reply:
Yeah adding the debian path would work too

Reply:
Done :+1: